File: Source/NasX/entity/qaobject.pas
unit QAObject;

{$mode objfpc}{$H+}

interface

uses
  Entity;

type
  { TQAObject }

  TQAObject = class(TEntity)
  private
    FAnswer: string;
    FQARule: string;
    FQuestion: string;
    FStoryId: Integer;
  published
    property Answer: string read FAnswer write FAnswer;
    property Id;
    property QARule: string read FQARule write FQARule;
    property Question: string read FQuestion write FQuestion;
    property StoryId: Integer read FStoryId write FStoryId;
  end;

implementation

{ TQAObject }

initialization
  TQAObject.RegisterEntityClass;

end.

File: Source/NeoAiSystems.com/NeoDesktop/Entity/QAObject.php
<?php
  namespace TApp;
  class TQAObject extends \sky\TEntity {
    protected function DefineProperties(){
      parent::DefineProperties();
      $this->AddProperties(array(
        "Answer" => array("check" => "string"),
        "Id" => array("check" => "Integer"),
        "StoryId" => array("check" => "Integer"),
        "QARule" => array("check" => "string"),
        "Question" => array("check" => "string")
      ));
    }
  }
?> 